HousingRepairsOnlineApi/Controllers/RepairController.cs

```diff
@@ -58,7 +58,11 @@ public async Task<IActionResult> CommunalPropertyRepairs([FromQuery] string prop
         [Route("TenantRepair")]
         public async Task<IActionResult> TenantRepair([FromBody] RepairRequest repairRequest)
         {
-            return await SaveRepair(RepairType.Tenant, repairRequest, bookAppointmentUseCase.Execute);
+            return await SaveRepair(RepairType.Tenant, repairRequest, BookAppointment);
+
+            Task BookAppointment(string bookingReference, string sorCode, string priority, string locationId,
```

Comment:
Is this code reached if we have a return statement above?

Reply:
This is a [local function](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/local-functions) and is used as the 3rd parameter to [SaveRepair on line 61](https://github.com/Newarkandsherwood/housing-repairs-online-api/pull/40/files#diff-2d7898aa4337b404555629732503b7d0b37f54f3384c9b388fb308e30d7a4fbdR61) (two lines above).

Reply:
Of course, I should have seen this.